tests/foreman/endtoend/test_api_endtoend.py

```diff
@@ -821,16 +822,24 @@ def test_positive_get_links(self):
         for group, path_pairs in api_paths.items():
             api_paths[group] = list(path_pairs.values())
 
-        if bz_bug_is_open(1166875):
-            # The server returns incorrect paths.
-            api_paths['docker_manifests'].append(u'/katello/api/docker_manifests')
-            api_paths['docker_manifests'].remove(u'/katello/api/compare')
-            api_paths['docker_tags'].append(u'/katello/api/docker_tags')
-            api_paths['docker_tags'].remove(u'/katello/api/compare')
-            api_paths['errata'].append(u'/katello/api/errata')
-            api_paths['errata'].append(u'/katello/api/errata/compare')
-            api_paths['errata'].remove(u'/katello/api/compare')
-
+            if bz_bug_is_open(1166875):
+                # The server returns incorrect paths.
+                if group == 'docker_manifests':
+                    new = list(path_pairs.values())
+                    new.remove(u'/katello/api/compare')
+                    new.insert(0, u'/katello/api/docker_manifests')
+                    api_paths[group] = new
```

Comment:
why we are creating a new copy of path_pairs.values()
perhaps the code should simplified 
```python 
api_paths[group].remove(u'/katello/api/compare')
api_paths[group].insert(0, u'/katello/api/docker_manifests')
```

Reply:
@ldjebran  api_paths[group] is dictionary having tuples as values so it shows 'dict' has no method 'insert'

Reply:
@ntkathole at line 823 we can read:
```python
api_paths[group] = list(path_pairs.values())
```
so we have already created that list copy

Reply:
@ldjebran cool..thanks..